include/workspaces.h

```diff
@@ -2,6 +2,9 @@
 #ifndef LABWC_WORKSPACES_H
 #define LABWC_WORKSPACES_H
 
+#include <stdbool.h>
+#include <wayland-util.h>
```

Comment:
Why `wayland-util.h` here? 
The `struct wl_list` is already forward declared.

Reply:
Forward declaration is not enough for the `struct wl_list link;` declaration below; the size of the struct needs to be known.

Reply:
We presumably don't see a compile error because in each case `wayland-util.h` is already included by some other header before `workspaces.h`.

Reply:
Sorry. Forward declaring is only enough for pointers. Ignore me :blush:
As I said... long day.